tests/Aspire.Azure.AI.OpenAI.Tests/Aspire.Azure.AI.OpenAI.Tests.csproj

```diff
@@ -6,7 +6,13 @@
 
   <ItemGroup>
     <None Include="$(RepoRoot)src\Components\Aspire.Azure.AI.OpenAI\ConfigurationSchema.json" CopyToOutputDirectory="PreserveNewest" />
+  </ItemGroup>
```

Comment:
nit: do we really need to split these into different item groups?

Reply:
This is how VS typically formats different items, and this separation makes it easier to reason about a project - one can clearly see project imports, package references, etc.